Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -13,8 +13,28 @@ def setup
   end
 
   def teardown
-    (HOMEBREW_PREFIX/"bin").rmtree
-    @formula_files.each(&:unlink)
```

Comment:
The only purpose of `@formula_files` is to keep track of what formula files need to be unlinked at teardown. If we don't use it any longer for that, we should get rid of it in `setup_test_formula`.